components/core/src/metrics.rs

```diff
@@ -37,27 +36,47 @@ enum Operation {
     Increment,
 }
 
-lazy_static! {
-    static ref SEND_CHANNEL : Mutex<Sender<(String, Operation)>> = {
-        let (tx, rx) : (Sender<(String, Operation)>, Receiver<(String, Operation)>) = channel();
-        let mut statsd_client = statsd_client();
-        thread::spawn(move || {
-            process_receives(rx, &mut statsd_client)
+// Helper type
+type CounterOp = (String, Operation);
+
+use std::sync::{Once, ONCE_INIT};
+
+static mut SENDER: *const Sender<CounterOp> = 0 as *const Sender<CounterOp>;
+
+static INIT: Once = ONCE_INIT;
+
+fn get_sender() -> Sender<CounterOp> {
```

Comment:
Love how you don't need to initialize and this function does that business for you if you haven't already